services-turf/src/main/java/com/mapbox/turf/TurfMisc.java

```diff
@@ -26,6 +26,43 @@ private TurfMisc() {
     throw new AssertionError("No Instances.");
   }
 
+  /**
+   * Takes lines {@link LineString} and returns intersect points.
```

Comment:
It'd be good to mention time complexity and that it's not as efficient as the JS counterpart.

Reply:
Thanks, I updated the comment